src/bundle/Resources/public/css/variables.css

```diff
@@ -0,0 +1,37 @@
+:root {
+    --ez-base-font-family: "Noto Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+
+    /* text colors */
+    --ez-color-text-base: #333;
+    --ez-color-text-white: #fff; /* should probably not be called text white */
```

Comment:
following custom properties are based on the names in http://ezsystems.github.io/ui-guidelines/#resources-colors but I think some names should be improved:

* *Text white* resulting in `--ez-color-text-white` should not include the actual color, if at some point we change it to something, *white* won't be white any longer :) I would suggest `--ez-color-text-inverted`
* Names that includes a component or a part of the UI should be changed, most likely the same color will be used in others contexts (maybe `--ez-color-hyperlink` can be an exception to that rule given usually this color is unique). The name should rather be what it means (neutral, primary, secondary are good for instance). I know, naming is hard ;-)

ping @SylvainGuittard @inakijv